### ISSUE TITLE
Series feature

### ISSUE THREAD

**Issue by duker33** [fidals/stroyprombeton#510]
This is rus text form messages. Translate it here and create pdd subtasks to start feature dev

**Data structure**
- [x] Create Series model. It's like category, but without hierarhy.
- [x] Series has it's own page.
- [x] Series is related to Product as 1:n ~or n:n. It depends on Series term meaning from domain area.~

**Sites navigation**
- [ ] All Series page. Looks as alphabetically sorted matrix of links on series pages. Has link from main menu.
- [ ] Series page contains products list. And all standard seo-stuff: text, meta tags, db templates and so on.
- [ ] May be Series page contain links on related categories.
- [ ] Product page has link on a related series.
- [ ] Category contains links on a related series. Link will lead on Series+Category page.
- [ ] Series+Category page is just a Series page in general. But contains breadcrums like category has. And it's products are filtered by related category.

```
страницы с сериями у меня есть в планах.
Вот что я хочу сделать в общих чертах:

Структура:
- выделяем отдельную сущность "Серия". Это ровно как Категория, но без иерархии
- в каждой серии лежат товары
- у одного товара мб нес-ко серий. Если я правильно понимаю что такое серия вообще

Навигация по сайту:
- в главном меню будет пункт Серии. И они там будут прям-таки матрицей. Можно будет искать через ctrl+f. Просто сделать, норм юзабилити, круто для перелинковки
- у каждой серии будет своя страница со списком товаров. С текстом, тегами, db templates и всяким таким
- на странице серии можно и список её категорий, если хочется. Будет похоже на крошки
- с товара будет линк на его серию
- в категории можно показывать серии, если хочется
```

**Comment by duker33** [comment]
take into acc categories with specification field. Some (or all) of these categories will become Series after script processing

**Comment by duker33** [comment]
checked on serv. Every product has one or zero related series
```
>>> tg = TagGroup.objects.get(name='Серия')
>>> for o in Option.objects.all():
...     count = o.tags.filter(group=tg).count()
...     if count > 1:
...             print(o.name, count)
...
```

**Comment by 0pdd** [comment]
@duker33 3 puzzles [#558](https://github.com/fidals/stroyprombeton/issues/558), [#559](https://github.com/fidals/stroyprombeton/issues/559), [#560](https://github.com/fidals/stroyprombeton/issues/560) are still not solved.

**Comment by 0pdd** [comment]
@duker33 5 puzzles [#558](https://github.com/fidals/stroyprombeton/issues/558), [#559](https://github.com/fidals/stroyprombeton/issues/559), [#568](https://github.com/fidals/stroyprombeton/issues/568), [#569](https://github.com/fidals/stroyprombeton/issues/569), [#570](https://github.com/fidals/stroyprombeton/issues/570) are still not solved; solved: [#560](https://github.com/fidals/stroyprombeton/issues/560).

**Comment by 0pdd** [comment]
@duker33 5 puzzles [#558](https://github.com/fidals/stroyprombeton/issues/558), [#559](https://github.com/fidals/stroyprombeton/issues/559), [#568](https://github.com/fidals/stroyprombeton/issues/568), [#569](https://github.com/fidals/stroyprombeton/issues/569), [#570](https://github.com/fidals/stroyprombeton/issues/570) are still not solved; solved: [#309](https://github.com/fidals/stroyprombeton/issues/309), [#560](https://github.com/fidals/stroyprombeton/issues/560).

**Comment by 0pdd** [comment]
@duker33 6 puzzles [#558](https://github.com/fidals/stroyprombeton/issues/558), [#559](https://github.com/fidals/stroyprombeton/issues/559), [#568](https://github.com/fidals/stroyprombeton/issues/568), [#583](https://github.com/fidals/stroyprombeton/issues/583), [#584](https://github.com/fidals/stroyprombeton/issues/584), [#585](https://github.com/fidals/stroyprombeton/issues/585) are still not solved; solved: [#309](https://github.com/fidals/stroyprombeton/issues/309), [#560](https://github.com/fidals/stroyprombeton/issues/560), [#569](https://github.com/fidals/stroyprombeton/issues/569), [#570](https://github.com/fidals/stroyprombeton/issues/570).

**Comment by 0pdd** [comment]
@duker33 5 puzzles [#558](https://github.com/fidals/stroyprombeton/issues/558), [#559](https://github.com/fidals/stroyprombeton/issues/559), [#583](https://github.com/fidals/stroyprombeton/issues/583), [#584](https://github.com/fidals/stroyprombeton/issues/584), [#585](https://github.com/fidals/stroyprombeton/issues/585) are still not solved; solved: [#309](https://github.com/fidals/stroyprombeton/issues/309), [#560](https://github.com/fidals/stroyprombeton/issues/560), [#568](https://github.com/fidals/stroyprombeton/issues/568), [#569](https://github.com/fidals/stroyprombeton/issues/569), [#570](https://github.com/fidals/stroyprombeton/issues/570).

**Comment by 0pdd** [comment]
@duker33 8 puzzles [#558](https://github.com/fidals/stroyprombeton/issues/558), [#559](https://github.com/fidals/stroyprombeton/issues/559), [#583](https://github.com/fidals/stroyprombeton/issues/583), [#584](https://github.com/fidals/stroyprombeton/issues/584), [#585](https://github.com/fidals/stroyprombeton/issues/585), [#599](https://github.com/fidals/stroyprombeton/issues/599), [#600](https://github.com/fidals/stroyprombeton/issues/600), [#601](https://github.com/fidals/stroyprombeton/issues/601) are still not solved; solved: [#309](https://github.com/fidals/stroyprombeton/issues/309), [#560](https://github.com/fidals/stroyprombeton/issues/560), [#568](https://github.com/fidals/stroyprombeton/issues/568), [#569](https://github.com/fidals/stroyprombeton/issues/569), [#570](https://github.com/fidals/stroyprombeton/issues/570).

**Comment by 0pdd** [comment]
@duker33 7 puzzles [#558](https://github.com/fidals/stroyprombeton/issues/558), [#559](https://github.com/fidals/stroyprombeton/issues/559), [#583](https://github.com/fidals/stroyprombeton/issues/583), [#584](https://github.com/fidals/stroyprombeton/issues/584), [#585](https://github.com/fidals/stroyprombeton/issues/585), [#599](https://github.com/fidals/stroyprombeton/issues/599), [#600](https://github.com/fidals/stroyprombeton/issues/600) are still not solved; solved: [#309](https://github.com/fidals/stroyprombeton/issues/309), [#560](https://github.com/fidals/stroyprombeton/issues/560), [#568](https://github.com/fidals/stroyprombeton/issues/568), [#569](https://github.com/fidals/stroyprombeton/issues/569), [#570](https://github.com/fidals/stroyprombeton/issues/570), [#601](https://github.com/fidals/stroyprombeton/issues/601).

**Comment by 0pdd** [comment]
@duker33 6 puzzles [#558](https://github.com/fidals/stroyprombeton/issues/558), [#559](https://github.com/fidals/stroyprombeton/issues/559), [#583](https://github.com/fidals/stroyprombeton/issues/583), [#584](https://github.com/fidals/stroyprombeton/issues/584), [#599](https://github.com/fidals/stroyprombeton/issues/599), [#600](https://github.com/fidals/stroyprombeton/issues/600) are still not solved; solved: [#309](https://github.com/fidals/stroyprombeton/issues/309), [#560](https://github.com/fidals/stroyprombeton/issues/560), [#568](https://github.com/fidals/stroyprombeton/issues/568), [#569](https://github.com/fidals/stroyprombeton/issues/569), [#570](https://github.com/fidals/stroyprombeton/issues/570), [#585](https://github.com/fidals/stroyprombeton/issues/585), [#601](https://github.com/fidals/stroyprombeton/issues/601).

**Comment by 0pdd** [comment]
@duker33 5 puzzles [#558](https://github.com/fidals/stroyprombeton/issues/558), [#559](https://github.com/fidals/stroyprombeton/issues/559), [#583](https://github.com/fidals/stroyprombeton/issues/583), [#584](https://github.com/fidals/stroyprombeton/issues/584), [#599](https://github.com/fidals/stroyprombeton/issues/599) are still not solved; solved: [#309](https://github.com/fidals/stroyprombeton/issues/309), [#560](https://github.com/fidals/stroyprombeton/issues/560), [#568](https://github.com/fidals/stroyprombeton/issues/568), [#569](https://github.com/fidals/stroyprombeton/issues/569), [#570](https://github.com/fidals/stroyprombeton/issues/570), [#585](https://github.com/fidals/stroyprombeton/issues/585), [#600](https://github.com/fidals/stroyprombeton/issues/600), [#601](https://github.com/fidals/stroyprombeton/issues/601).

**Comment by 0pdd** [comment]
@duker33 3 puzzles [#583](https://github.com/fidals/stroyprombeton/issues/583), [#599](https://github.com/fidals/stroyprombeton/issues/599), [#709](https://github.com/fidals/stroyprombeton/issues/709) are still not solved; solved: [#309](https://github.com/fidals/stroyprombeton/issues/309), [#558](https://github.com/fidals/stroyprombeton/issues/558), [#559](https://github.com/fidals/stroyprombeton/issues/559), [#560](https://github.com/fidals/stroyprombeton/issues/560), [#568](https://github.com/fidals/stroyprombeton/issues/568), [#569](https://github.com/fidals/stroyprombeton/issues/569), [#570](https://github.com/fidals/stroyprombeton/issues/570), [#584](https://github.com/fidals/stroyprombeton/issues/584), [#585](https://github.com/fidals/stroyprombeton/issues/585), [#600](https://github.com/fidals/stroyprombeton/issues/600), [#601](https://github.com/fidals/stroyprombeton/issues/601).

**Comment by 0pdd** [comment]
@duker33 2 puzzles [#583](https://github.com/fidals/stroyprombeton/issues/583), [#599](https://github.com/fidals/stroyprombeton/issues/599) are still not solved; solved: [#309](https://github.com/fidals/stroyprombeton/issues/309), [#558](https://github.com/fidals/stroyprombeton/issues/558), [#559](https://github.com/fidals/stroyprombeton/issues/559), [#560](https://github.com/fidals/stroyprombeton/issues/560), [#568](https://github.com/fidals/stroyprombeton/issues/568), [#569](https://github.com/fidals/stroyprombeton/issues/569), [#570](https://github.com/fidals/stroyprombeton/issues/570), [#584](https://github.com/fidals/stroyprombeton/issues/584), [#585](https://github.com/fidals/stroyprombeton/issues/585), [#600](https://github.com/fidals/stroyprombeton/issues/600), [#601](https://github.com/fidals/stroyprombeton/issues/601), [#709](https://github.com/fidals/stroyprombeton/issues/709).
